src/main/java/de/tum/in/www1/artemis/web/rest/programming/ProgrammingExerciseExportImportResource.java

```diff
@@ -230,15 +230,6 @@ public ResponseEntity<ProgrammingExercise> importProgrammingExercise(@PathVariab
         Course originalCourse = courseService.retrieveCourseOverExerciseGroupOrCourseId(originalProgrammingExercise);
         authCheckService.checkHasAtLeastRoleInCourseElseThrow(Role.EDITOR, originalCourse, user);
 
-        // Athena: Check that only allowed athena modules are used, if not we catch the exception and disable feedback suggestions for the imported exercise
-        // If Athena is disabled and the service is not present, we also disable feedback suggestions
-        try {
-            athenaModuleService.ifPresentOrElse(ams -> ams.checkHasAccessToAthenaModule(newExercise, course, ENTITY_NAME), () -> newExercise.setFeedbackSuggestionModule(null));
```

Comment:
do you remove this code intentionally? Or is it a mistake?

@maximiliansoelch would this have any impact on Athena functionality?

Reply:
Yes, as it would allow unrestricted access to normally restricted Athena modules (LLM Modules) when importing a programming exercise and also would probably throw some errors when a programming exercise is imported from another Athena instance where Athena is enabled to an instance where it is disabled.

Reply:
Curious how this happened 🤔 

I'll revert it.